bio/gatk/printreadsspark/meta.yaml

```
@@ -0,0 +1,18 @@
+name: gatk PrintReadsSpark
+description: |
+  Run gatk PrintReadsSpark.
```

Maybe make this describe what PrintReadsSpark actually does?
```suggestion
  Write reads from SAM format file (SAM/BAM/CRAM) that pass specified criteria to a new file. This is the version that can be run on Spark.
```